config/locales/en.yml

```diff
@@ -171,6 +171,8 @@ en:
           true: Serial
         subtitle: Teaser
         title: Title
+      podcast_planner:
+        selected_days: asanoteuh
```

Comment:
where does this show up?

Reply:
Whoops!  Leftover from some testing.  I think I was trying to get rails to do its magical translation of model fieldnames.  But I never did get it working.